lib/iron_bank/local_records.rb

```diff
@@ -6,52 +6,91 @@ module IronBank
   class LocalRecords
     private_class_method :new
 
-    RESOURCES = %w[
-      Product
-      ProductRatePlan
-      ProductRatePlanCharge
-      ProductRatePlanChargeTier
-    ].freeze
+    RESOURCE_QUERY_FIELDS = {
+      "Product"                   => ["*"],
+      "ProductRatePlan"           => ["*", "Product.Id"],
+      "ProductRatePlanCharge"     => ["*", "ProductRatePlan.Id"],
+      "ProductRatePlanChargeTier" => ["*", "ProductRatePlanCharge.Id"]
+    }.freeze
 
     def self.directory
       IronBank.configuration.export_directory
     end
 
     def self.export
       FileUtils.mkdir_p(directory) unless Dir.exist?(directory)
-      RESOURCES.each { |resource| new(resource).export }
+      RESOURCE_QUERY_FIELDS.keys.each { |resource| new(resource).save_file }
     end
 
-    def export
-      CSV.open(file_path, "w") do |csv|
-        # first row = CSV headers
-        write_headers(csv)
-        write_records(csv)
+    def save_file
+      until completed? || max_query?
+        IronBank.logger.info(export_query_info)
+        sleep backoff_time
       end
+
+      File.open(file_path, "w") { |file| file.write(export.content) }
     end
 
     private
 
-    attr_reader :resource
+    BACKOFF = {
```

Comment:
WYDTA removing some redundancy?
```suggestion
    BACKOFF = {
      max:        3,
      interval:   0.5,
      randomness: 0.5,
      factor:     4
    }.freeze 
```